.circleci/config.yml

```diff
@@ -2,34 +2,24 @@ version: 2
 
 references:
   container_config: &container_config
-    setup_remote_docker:
-      docker_layer_caching: true
     docker:
-      - image: aeternity/builder
-        user: builder
+      - image: aeternity/infrastructure:master
```

Comment:
I see this PR is a chicken-and-egg one - introducing both the build+push of aeternity/infrastructure:master Docker image *and its usage*. So this can only work if this PR is merged well before the time of trigger of setup of environments (midday UTC ? every day so merge shall happen e.g. in afternoon or in very early morning - otherwise a setup env run would fail).

Reply:
I'm not sure I understand your concern about timings, the cron auto-setup will run whatever i sin `aeternity/infrastructure:master` image. The configuration that it runs on master is kind of misleading, because it actually does not care of the current repo source code, but the latest `aeternity/infrastructure:master` which is in fact the latest master.